monitoring_flows_and_extensions/index.md

```diff
@@ -0,0 +1,335 @@
+	overview: ({ to, events }) => [
+		{
+			label: 'To',
+			text: Array.isArray(to) ? to.join(', ') : to,
+		},
+		{
+			label: 'Events',
+			text: Array.isArray(events) ? to.join(', ') : events,
+		}
+	],
+	options: [
+		{
+			field: 'to',
+			name: 'To',
+			type: 'csv',
+			meta: {
+				width: 'full',
+				interface: 'tags',
+				options: {
+					placeholder: '$t:operations.mail.to_placeholder',
+					iconRight: 'alternate_email',
+				},
+			},
+		},
+		{
+			field: 'events',
+			name: 'Events',
+			type: 'csv',
+			meta: {
+				width: 'full',
+				interface: 'tags',
+				options: {
+					placeholder: 'Nach jedem Event mit ENTER bestätigen',
+				},
+			},
+		},
+	],
+});
+
+```
+</details>
+
+<details>
+<summary>Show api definition of notify_on_event</summary>
+
+```ts
+import { defineOperationApi } from '@directus/extensions-sdk';
+
+type Options = {
+	to: string[];
+	events: string[];
+};
+
+export default defineOperationApi<Options>({
+	id: 'notify_on_event',
+	handler: async ({ to, events }, { getSchema, database, services, accountability, logger, data }) => {
+		const trigger: any = data.$trigger;
+
+		const payload = trigger.payload;
+		if (!events.includes(payload.event_name)) {
+			return;
+		}
+
+		payload.meta = liquifyMeta(payload.meta);
+
+		const schema = await getSchema({ database });
+		const context = {database: database, schema: schema, accountability: accountability};
+
+		const { MailService } = services;
+        const mailService = new MailService({ schema: context.schema, knex: context.database });
+        
+		await mailService.send({
+			to: to,
+			subject: `Directus Event - ${payload.event_name}`,
+			template: {
+				name: 'event-notification',
+				data: payload,
+			},
+		});
+	},
+});
+
+function liquifyMeta(meta: Object) {
+	const ret = [];
+	for (const [key, value] of Object.entries(meta)) {
+		ret.push({'key': key, 'value': JSON.stringify(value)});
+	}
+	return ret;
+}
+
+```
+</details>
+
+<details>
+<summary>Email template used in notify_on_event</summary>
+
+```html
+<body>
+  Eventname: {{ event_name }}<br>
+  Eventdate: {{ event_date }}<br>
+  Context: {{ context }}<br>
+  <p>
+    Meta:<br>
+    {% for metadata in meta %}
+      {{ metadata.key }}: {{ metadata.value }}<br>
+    {% endfor %}
+  </p>
+  Execution ID: {{ execution_id }}
+</body>
+```
+</details>
+
+## Summary
+While this setup is straightforward, it serves as a foundational step for monitoring your application. Typically, you would externalize your monitoring to avoid impacting your production database. However, if your workload is relatively small, this approach remains a viable solution, as you won't require any third-party services.
+
+With this configuration in place, you have the option to create an insights panel to enhance your data analysis. Additionally, you can revamp the notification process by customizing the default email template or adding more templates based on the event's name.
```

Comment:
```suggestion
This setup serves as a foundational step for monitoring your application. Typically, you would externalize your monitoring to avoid impacting your production database. However, if your workload is relatively small, this approach is a viable solution, as you won't require any third-party services.

With this configuration in place, you have the option to create an insights panel to enhance your data analysis. Additionally, you can revamp the notification process by customizing the default email template or adding more templates based on the event's name.
```